mirage/factories/partnerclient-plan.js

```diff
@@ -0,0 +1,18 @@
+import {
```

Comment:
Can you keep partner related factories inside partner folder?

Reply:
done